src/NServiceBus.Core/obsoletes-v8.cs

```diff
@@ -806,4 +806,44 @@ public interface ICancelDeferredMessages
     }
 }
 
+namespace NServiceBus.MessageMutator
+{
+    using System.Threading.Tasks;
+
+    [ObsoleteEx(
+        Message = "Use behaviors instead",
+        TreatAsErrorFromVersion = "8.0.0",
```

Comment:
Should be `TreatAsErrorFromVersion = "9.0.0",` to give a WARN in v8 right?

Reply:
👍 we'll follow whatever we agree on in https://github.com/Particular/Platform/pull/10

Reply:
👍 we agreed to WARN in NServiceBus 8, ERROR in NServiceBus 9, and remove in NServiceBus 10.